techniques/failures/F96.html

```diff
@@ -2,148 +2,114 @@
-               text labelling some input). If the visible label does not match the accessible name
-               of the control, speech users may be unable to directly activate that control.
-            </p>
-            <p>There are techniques that go beyond the visible label text of elements. The idea is
-               to provide more context for users of assistive technologies to whom the visible context
-               may not be available. Examples are the use of accessible hidden (e.g., offscreen-positioned)
-               text, the use of the aria-label attribute to overwrite the label text, or the use
-               of the aria-labelledby attribute to compose a label text from one or more strings
-               strings on the page.
-            </p>
-            <p>When the use of these techniques results in an accessible name in which the exact
-               string of the visible label does not occur in the accessible name, speech users may
-               be unable to activate that control. Refer to the <a href="https://www.w3.org/TR/accname/">Accessible Name and Description Computation</a> algorithm to work out the order of precedence in computing the accessible name of
-               a control.
-            </p>
+            <h2>解説</h2>
+            <p>この失敗例の目的は、コントロールの名前と、目に見えるラベルが異なっているために、音声入力のユーザーがコントロールの名前を正しく話せないという状況を説明することである。</p>
+            <p>音声入力のユーザは、Webページを操作するとき、通常はコマンドの後に何らかの表示ラベル(ボタンやリンクの中のテキストや、入力欄にラベリングしてあるテキスト)への参照を続けて話す。（永野注: 日本語でも”コマンド→対象”の順で話すのかな？）表示されるラベルが、コントロールのアクセシブルな名前と一致しない場合、音声を使うユーザーは、そのコントロールを直接アクティブにできないことがある。</p>
+            <p>要素のラベルテキストを可視化するのではなく、それを超える方法がある。これは、視覚的な文脈を利用できない支援技術の利用者に、より多くの文脈を提供するためのアイデアである。たとえば、非表示でもアクセス可能なテキスト(例: 表示領域以外に配置したテキスト)の使用や、ラベルテキストを上書きするaria-label属性の使用、または、ページ上の1つ以上の文字列からラベルテキストを作成するaria-labeledby属性の使用があります。</p>
+            <p>これらの方法を使用した結果、表示されているラベルの正確な文字列が、アクセシブルな名前に含まれない場合、音声を使うユーザはそのコントロールをアクティブにできない可能性があります。コントロールのアクセシブルな名前を計算する際の優先順位を決定するには、 <a href="https://www.w3.org/TR/accname/">「アクセス可能な名前と説明の計算」</a>アルゴリズムを参照してください。</p>
          </section>
          <section id="examples">
-            <h2>Examples</h2>
+            <h2>事例</h2>
             <section class="example" id="example-1-mismatch-of-visible-button-text-and-accessible-name-supplied-via-aria-label">
-               <h3>Example 1: Mismatch of visible button text and accessible name supplied via aria-label</h3>
-               <p>The text in a search button reads "Go" but the accessible name provided in an aria-label
-                  attribute is "Find in this site"
-               </p><pre><code>
-        &lt;button id="sitesearch" aria-label="Find in this site"&gt;Go&lt;/button&gt;
-        </code></pre></section>
+               <h3>失敗例1: 表示されているボタンのテキストと、aria-labelで指定されたアクセシブルな名前が一致しない</h3>
+               <p>検索ボタンのテキストは「OK」ですが、aria-label属性で提供されるアクセシブルな名前は、「このサイトで検索」です。</p>
+               <pre><code>&lt;button id="sitesearch" aria-label="このサイトで検索"&gt;OK&lt;/button&gt;</code></pre>
```

Comment:
```suggestion
               <pre><code>&lt;button id="sitesearch" aria-label="Find in this site"&gt;Go&lt;/button&gt;</code></pre>
```

コード例は翻訳しません。例2と3についてもその方針で見直してください。